frontend/lib/src/components/elements/DeckGlJsonChart/DeckGlJsonChart.tsx

```diff
@@ -54,20 +55,21 @@ const EMPTY_LAYERS: LayersList = []
 
 export const DeckGlJsonChart: FC<DeckGLProps> = props => {
   const {
-    collapse,
     disabled,
     disableFullscreenMode,
     element,
-    expand,
     fragmentId,
-    height: propsHeight,
-    isFullScreen,
     mapboxToken: propsMapboxToken,
     widgetMgr,
-    width: propsWidth,
   } = props
   const { mapboxToken: elementMapboxToken } = element
   const theme: EmotionTheme = useTheme()
+  const {
+    expanded: isFullScreen,
```

Comment:
fwiw, i like isFullScreen much more than expanded 🤷

Reply:
I was using the same naming convention [as existed here](https://github.com/streamlit/streamlit/blob/develop/frontend/lib/src/components/shared/FullScreenWrapper/FullScreenWrapper.tsx#L53). But feel free to change it up!

Reply:
`isFullScreen` also looks a bit more descriptive if used in an element. But no strong opinion.